src/pivotal_api_services/pivotal_services.py

```diff
@@ -6,4 +6,5 @@ class PivotalServices:
     __metaclass__ = ABCMeta
 
     def __init__(self):
-        self.request_handler = RequestManager.get_instance()
+        #self.request_handler = RequestManager.get_instance()
+        self.request_handler = RequestManager.get_instanceA()
```

Comment:
this is going to give problems to you teammates, keep both variables and just change the variable name of the second